c51686645.lua

```diff
@@ -89,12 +89,15 @@ function c51686645.eqlimit(e,c)
 	return c:IsControler(1-e:GetHandlerPlayer())
 end
 function c51686645.drcon(e,tp,eg,ep,ev,re,r,rp)
-	return e:GetHandler():IsReason(REASON_LOST_TARGET)
-		and not e:GetHandler():GetPreviousEquipTarget():IsLocation(LOCATION_ONFIELD+LOCATION_OVERLAY)
+	if e:GetHandler():IsReason(REASON_LOST_TARGET)
+		and not e:GetHandler():GetPreviousEquipTarget():IsLocation(LOCATION_ONFIELD+LOCATION_OVERLAY) then return true end
+	e:GetHandler():ResetFlagEffect(51686645)
```

Comment:
why reset here?

Reply:
'Tis a security measure in case the card goes to the GY without its effect triggering, e.g. if the conditions for it aren't met.

Reply:
it will always be reset when the card leave grave and can trigger the 2nd effect again

Reply:
Okay, but how many of the specified flag effect will `Card.ResetFlagEffect` remove?